tests/test_math_ops_coverage.py

```
@@ -0,0 +1,854 @@
+        ##################################################################################
+        # Test _div_derivs edge cases
+        ##################################################################################
+        # Test with nozeros=False (line 774-775)
+        a = Scalar([1., 2., 3.])
+        a.insert_deriv('t', Scalar([0.1, 0.2, 0.3]))
+        b = Scalar([2., 0., 4.])
+        b.insert_deriv('t', Scalar([0.4, 0.5, 0.6]))
+        # This will call _div_derivs internally through division
+        try:
+            c = a / b
+        except:
```

Ruff (0.14.7)

Do not use bare `except`

(E722)